src/components/timepicker/timePicker.js

```diff
@@ -0,0 +1,53 @@
+import 'react-widgets/styles.css';
+import DatePicker from 'react-widgets/DatePicker';
+import TimeInput from 'react-widgets/TimeInput';
+
+import { useState } from 'react';
+
+function dateToEpoch (date) {
+    const timeInEpochSeconds = Date.parse(date)
+
+    // return epoch time in seconds
+    return timeInEpochSeconds / 1000
+}
+
+// Datetime input using 
+// React Widgets DatePicker and TimePicker
+function TimePicker(props) {
+    const [dateTime, setDateTime] = useState(new Date())
+    const [epochTime, setEpochTime] = useState(dateToEpoch(new Date()))
+
+    const updateTime = (newTime) => {
+        if (newTime == null || isNaN(newTime)) {
+            // handles invalid input from datepicker and timeinput
+            setDateTime(new Date())
+            setEpochTime(dateToEpoch(new Date()))
+        }
+        else {
+            const timeToEpoch = dateToEpoch(newTime);
+            setDateTime(newTime)
+            setEpochTime(timeToEpoch)
+            props.setParentTimeInput(timeToEpoch)
+        }
+    }
+
+    return (
+        <>
```

Comment:
Would it be better to use a div here or is an empty tag best practice? I've never that before

Reply:
It's syntactic sugar for React fragment (https://reactjs.org/docs/fragments.html) so instead of adding another div to contain the DatePicker and TimePicker components we add them directly to the parent. I'm pretty sure either way works tho